@xen-orchestra/fs/src/_copyStreamToBuffer.test.js

```diff
@@ -1,7 +1,8 @@
-/* eslint-env jest */
+const { describe, it } = require('test')
+const assert = require('assert').strict
 
-import { Readable } from 'readable-stream'
-import copyStreamToBuffer from './_copyStreamToBuffer.js'
+const { Readable } = require('readable-stream')
+const copyStreamToBuffer = require('./../dist/_copyStreamToBuffer.js').default
```

Comment:
Because I can't import it from ```./_copyStreamToBuffer.js```, otherwise I get an error about the ```export default``` of the function.

I saw you did something similar here: https://github.com/vatesfr/xen-orchestra/blob/master/packages/xen-api/_parseUrl.spec.js#LL5C56-L5C56

Reply:
Yes, because `_parseUrl.spec.js` is not in `src/` and is therefore not compiled and cannot use `src/_parseUrl.js` directly.

You should not use this here.